package.json

```diff
@@ -30,6 +30,7 @@
     "chalk": "^5.2.0",
     "child_process": "^1.0.2",
     "cli-table": "^0.3.11",
-    "commander": "^10.0.0"
+    "commander": "^10.0.0",
+    "json2csv": "^6.0.0-alpha.2"
```

Comment:
Why are we using an alpha version here? Can't we move to a more stable version? Why not 6.0.0 or 7.0.6? https://www.npmjs.com/package/@json2csv/node?activeTab=versions